naga/Cargo.toml

```diff
@@ -72,6 +72,7 @@ termcolor = { version = "1.4.1" }
 # termcolor minimum version was wrong and was fixed in
 # https://github.com/brendanzab/codespan/commit/e99c867339a877731437e7ee6a903a3d03b5439e
 codespan-reporting = { version = "0.11.0" }
+hashbrown = { workspace = true, features = ["serde"] }
```

Comment:
`serde` should be enabled only via `serialize`/`deserialize` features. If there's something that needs serialization without those features that would be a bit fishy.